flocker/testtools/test/test_base.py

```diff
@@ -11,7 +11,8 @@
 from eliot import MessageType, fields
 from hypothesis import assume, given
 from hypothesis.strategies import binary, integers, lists, text
-from testtools import PlaceHolder, TestCase, TestResult
+from testtools import PlaceHolder, TestResult
+from testtools import TestCase as TesttoolsTestCase
```

Comment:
The reason why we use this might warrant a comment. It took staring at the code to realize that we are
- using this to test that our testcase subclasses behave sanely
- don't want to use the subclasses themselves, in case they are buggy